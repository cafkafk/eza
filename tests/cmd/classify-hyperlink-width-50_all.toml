bin.name = "eza"
args = "tests/itest/ --width 50 --classify --hyperlink"
