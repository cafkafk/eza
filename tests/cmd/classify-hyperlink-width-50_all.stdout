]8;;file:///home/ces/org/src/git/eza/tests/itest/a/a]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/exa/exa]8;;//  ]8;;file:///home/ces/org/src/git/eza/tests/itest/image.jpg.img.c.rs.log.png/image.jpg.img.c.rs.log.png]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/m/m]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/vagrant/vagrant]8;;//
]8;;file:///home/ces/org/src/git/eza/tests/itest/b/b]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/f/f]8;;/     ]8;;file:///home/ces/org/src/git/eza/tests/itest/index.svg/index.svg]8;;/                   ]8;;file:///home/ces/org/src/git/eza/tests/itest/n/n]8;;/  
]8;;file:///home/ces/org/src/git/eza/tests/itest/c/c]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/g/g]8;;/     ]8;;file:///home/ces/org/src/git/eza/tests/itest/j/j]8;;/                           ]8;;file:///home/ces/org/src/git/eza/tests/itest/o/o]8;;/  
]8;;file:///home/ces/org/src/git/eza/tests/itest/d/d]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/h/h]8;;/     ]8;;file:///home/ces/org/src/git/eza/tests/itest/k/k]8;;/                           ]8;;file:///home/ces/org/src/git/eza/tests/itest/p/p]8;;/  
]8;;file:///home/ces/org/src/git/eza/tests/itest/e/e]8;;/  ]8;;file:///home/ces/org/src/git/eza/tests/itest/i/i]8;;/     ]8;;file:///home/ces/org/src/git/eza/tests/itest/l/l]8;;/                           ]8;;file:///home/ces/org/src/git/eza/tests/itest/q/q]8;;/  
